torch/csrc/jit/tensorexpr/ir_simplifier.cpp

```diff
@@ -1077,6 +1077,27 @@ Stmt* PolynomialTransformer::mutate(const For* v) {
   return new For(var_new, start_new, stop_new, body_new, loop_options);
 }
 
+Stmt* PolynomialTransformer::mutate(const Block* v) {
+  std::vector<Stmt*> stmts;
+  for (Stmt* stmt : v->stmts()) {
+    Stmt* stmt_new = stmt->accept_mutator(this);
+    if (!stmt_new) {
+      continue;
```

Comment:
Hm, wouldn't we leave out unchanged statements due to this logic?

Reply:
No, we filter out null statements here. I'll adjust the condition to explicitly check against nullptr.